src/frontend/js/translations/en-US.json

```diff
@@ -184,6 +184,8 @@
   "components.ReferralForm.RequesterField.label": "Requester name",
   "components.ReferralForm.TopicField.description": "Broad topic to help direct the referral to the appropriate office",
   "components.ReferralForm.TopicField.loadingTopics": "Loading topics...",
+  "components.ReferralForm.TopicFields.noItemsfound": "",
```

Comment:
For english strings you just need to copy-paste them in Poedit so they appear in the translations file.